test/integration/full/landmark-one-main/landmark-one-main-fail.js

```diff
@@ -0,0 +1,34 @@
+describe('landmark-one-main test failure', function () {
+	'use strict';
+	var results;
+	before(function (done) {
+		axe.run({ runOnly: { type: 'rule', values: ['landmark-one-main'] } }, function (err, r) {
+			assert.isNull(err);
+			results = r;
+			done();
+		});
+	});
+
+	describe('violations', function () {
+		it('should find 1', function () {
+			assert.lengthOf(results.violations, 1);
```

Comment:
Check the number of nodes. I think that should be 2. You're not testing the second node here either.

Reply:
@WilcoFiers @jongund 
It appears as if results.violations only has one node even though it should be observing two violations. We couldn't find any implementation of a test for more than one violation. 
However, the testing results.passes in landmark-one-main-pass.js does return 4 nodes.
Any thoughts on why violations only seems to contain one node, for our test as well as the other tests in the aXe library?

Reply:
@ssanaul Looks like another timing issue. You're calling axe.run before the frames are loaded. Have a look at how that was solved in landmark-main-is-top-level in the before.

Reply:
@WilcoFiers @jongund 
Both nodes are tested now.

Reply:
Excellent. I think that will do it!! Thanks @ssanaul!